portal/src/texts/blog/six-advice-on-mixpanel.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "6 Advice on Mixpanel Integration"
+author: Sercan Leylek
+description: Tips for teams which will include Mixpanel to their web applications.
+publishDate: "2022.03.12"
+---
+
+<Ingress>
+    Mixpanel is a business analytics service which tracks user interactions on web and mobile applications. Integrating
+    a tracking analytics tool with an application may seem like a simple task, but this story recommends to review your
+    plan once more.
+</Ingress>
+
+User analytics is one of the most fundamental features that every popular web application should perform.
+Without collecting data about user activities, one can never be sure of the user experience.
+The developer team will invest days on some functionality, but unless they know how many users use this precious feature,
+they will only guess and hope that the users are benefiting from it.
+In other words, “Without data you’re just another person with an opinion.” W. Edwards Deming.
```

Comment:
Har du en lenke til hvor dette sitatet kommer fra? Jeg kjenner ikke til personen eller sitatet 😅

Reply:
Jeg har ingen lenke til å verifisere det akkurat. Men ETF har også brukt det :) https://www.etf.europa.eu/en/news-and-events/news/without-data-youre-just-another-person-opinion-w-edwards-deming